internal/querycoordv2/checkers/checker.go

```diff
@@ -0,0 +1,26 @@
+package checkers
+
+import (
+	"context"
+
+	"github.com/milvus-io/milvus/internal/querycoordv2/task"
+)
+
+type Checker interface {
+	ID() int64
+	SetID(id int64)
+	Description() string
+	Check(ctx context.Context) []task.Task
```

Comment:
it only generates checker tasks, so it should not be named Check?

Reply:
> 

It makes sense that the Check function generates tasks.